6.1/Dockerfile

```diff
@@ -51,4 +51,5 @@ RUN echo '/usr/local/lib64' > /etc/ld.so.conf.d/local-lib64.conf \
 RUN set -x \
 	&& dpkg-divert --divert /usr/bin/gcc.orig --rename /usr/bin/gcc \
 	&& dpkg-divert --divert /usr/bin/g++.orig --rename /usr/bin/g++ \
+	&& dpkg-divert --divert /usr/bin/gfortran.orig --rename /usr/bin/gfortran \
```

Comment:
I don't think so. I'm not familiar with the goals of buildpack-deps or whether it might add gfortran in the future, so it seemed safe to add just in case.